basic-pr-verification/action.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: 'Basic PR verification'
```

Comment:
Let's rename it everywhere (including directory name) to "PR verification". It is basic now, but it will have more, and I'm thinking about adding to instead of creating separate actions to make workflows faster.

Also, a long name might be shortened by GitHub UI.